src/com/owncloud/android/authentication/AuthenticatorActivity.java

```diff
@@ -1974,11 +1985,13 @@ public void createAuthenticationDialog(WebView webView, HttpAuthHandler handler)
         dialog.show(ft, CREDENTIALS_DIALOG_TAG);
 
         if (!mIsFirstAuthAttempt) {
-            Toast.makeText(
-                    getApplicationContext(), 
-                    getText(R.string.saml_authentication_wrong_pass), 
-                    Toast.LENGTH_LONG
-            ).show();
+            Snackbar snackbar = Snackbar.make(

```

Comment:
You can create a new method to show the snackbar, with the error message as parameter.